docs/user/translating.rst

```diff
@@ -36,8 +36,8 @@ Translation links
 
 Having navigated to a component, a set of links lead to its actual translation.
 The translation is further divided into individual checks, like
-:guilabel:`Not translated strings` or :guilabel:`Strings needing action`. If the whole project
-is translated, without error, :guilabel:`All strings` is still available.
+:guilabel:`Not translated strings` or :guilabel:`Strings needing action`.
+If the whole project is translated, without error, :guilabel:`All strings` is still available.
```

Comment:
```suggestion
If the whole project is translated without any errors, :guilabel:`All strings` is still available.
```
This actually meant the label is always available when a project is translated, as opposed to some of the time.